rsocket-core/src/main/java/io/rsocket/RSocketFactory.java

```diff
@@ -99,9 +100,9 @@ static ServerRSocketFactory receive() {
   }
 
   interface MimeType<T> {
-    T mimeType(String dataMineType, String metadataMimeType);
+    T mimeType(String metadataMimeType, String dataMimeType);
```

Comment:
As all other places use this new order, I also believe it should be ..(metadataMimeType, dataMimeType)